src/transformers/models/luke/tokenization_luke.py

```diff
@@ -189,75 +251,55 @@ def __init__(
         self,
         vocab_file,
         merges_file,
-        entity_vocab_file,
-        task=None,
-        max_entity_length=32,
-        max_mention_length=30,
-        entity_token_1="<ent>",
-        entity_token_2="<ent2>",
-        entity_unk_token="[UNK]",
-        entity_pad_token="[PAD]",
-        entity_mask_token="[MASK]",
-        entity_mask2_token="[MASK2]",
+        errors="replace",
+        bos_token="<s>",
+        eos_token="</s>",
+        sep_token="</s>",
+        cls_token="<s>",
+        unk_token="<unk>",
+        pad_token="<pad>",
+        mask_token="<mask>",
+        add_prefix_space=False,
         **kwargs
     ):
-        # we add 2 special tokens for downstream tasks
-        # for more information about lstrip and rstrip, see https://github.com/huggingface/transformers/pull/2778
-        entity_token_1 = (
-            AddedToken(entity_token_1, lstrip=False, rstrip=False)
-            if isinstance(entity_token_1, str)
-            else entity_token_1
-        )
-        entity_token_2 = (
-            AddedToken(entity_token_2, lstrip=False, rstrip=False)
-            if isinstance(entity_token_2, str)
-            else entity_token_2
-        )
-        kwargs["additional_special_tokens"] = kwargs.get("additional_special_tokens", [])
-        kwargs["additional_special_tokens"] += [entity_token_1, entity_token_2]
```

Comment:
Same, this specific behavior should be left as it's something on top of the Roberta tokenizer init.